vgen/src/main/resources/io/gapi/vgen/go/example.snip

```diff
@@ -0,0 +1,46 @@
+@snippet generateFilename(service)
+    {@context.upperCamelToLowerUnderscore(service.getSimpleName)}_test/client_test.go
+@end
+
+@snippet generateClass(service, body)
+    package {@context.upperCamelToLowerUnderscore(service.getSimpleName)}
+
+    {@importSection(service)}
+
+    {@body}
+@end
+
+@private importSection(service)
+    import (
+    @join import : context.getTestImports(service)
+    {@import}
+    @end
+    )
+@end
+
+@snippet generateBody(service)
+    func ExampleNewClient() {
+        ctx := context.Background()
+        opts := []gax.ClientOption{ /* Optional client parameters. */ }
+        _, _ = {@context.upperCamelToLowerUnderscore(service.getSimpleName)}.NewClient(ctx, opts...)
+    }
+    @join method : service.getMethods
+        @let methodName = method.getSimpleName, \
+             inTypeName = context.messageTypeName(method.getInputType), \
+             outTypeName = context.typeName(method.getOutputType), \
+             methodConfig = context.getApiConfig.getInterfaceConfig(service).getMethodConfig(method), \
+             isPageStreaming = methodConfig.isPageStreaming
```

Comment:
Are we doing anything with this? Do we want to give example of a loop if the method page-streams?